simphony/io/h5_lattice.py

```diff
@@ -28,20 +29,22 @@ def __init__(self, group):
             raise ValueError("Lattice file layout has an incompatible version")
 
         self._group = group
-        self._type = group.lattice.attrs.type
-        self._base_vect = group.lattice.attrs.base_vect
+        self._prim_cell = PrimitiveCell(group.lattice.attrs.prim_cell[0],
+                                        group.lattice.attrs.prim_cell[1],
+                                        group.lattice.attrs.prim_cell[2])
+        self._prim_cell._bravais_lattice = BravaisLattice(
```

Comment:
In order to avoid using the non-public `_bravais_lattice`  attribute here, I would suggest possibly reworking PrimitiveCell so that `bravais_lattice` is also a parameter in PrimitiveCell's ` __init__`